go.mod

```diff
@@ -265,7 +265,7 @@ require (
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
 
-replace github.com/ethereum/go-ethereum v1.13.11 => github.com/ethereum-optimism/op-geth v1.101311.0-rc.1
+replace github.com/ethereum/go-ethereum v1.13.11 => github.com/ethereum-optimism/op-geth v1.101311.0-rc.1.0.20240405183842-fbf1ff74c288
```

Comment:
Commit LGTM. It may be nice to cut a new patch release of op-geth

Reply:
Turns out the fix is in the final `v1.101311.0` - it just never got pulled into the monorepo so updated this to point to that release instead of -rc.1.